### ISSUE TITLE
No character description in Notepad++

### ISSUE THREAD

**Issue by DGartmann** [davidacm/EnhancedPhoneticReading#2]
Using a OneCore voice eg Hazel, I am using Left or Right arrow to move character by character over a couple of characters both small and capital letters. I get no character descriptions after hearing the character name itself. I am nusing the 1.1.1 dev version of the add-on. In standard Notepad on my Windows10 machine, the descriptions are read out fine after the delay I have configred.

**Comment by DGartmann** [comment]
Addition: I am using NVDA 2021.3.4 and Notepad++ v8.2.1(64-bit)

**Comment by davidacm** [comment]
Hi. Are you using a notepad++ profile? and... Are you sure that "Enable delayed descriptions for characters" is enabled in notepad++ profile?

To check it, open Notepad++, and then NVDA options, then see if "Enable delayed descriptions for characters" is enabled in "Enhanced phonetic reading" category.

I tried with the same version of NVDA and notepad++, but all worked without issues.

**Comment by DGartmann** [comment]
Yes. I am using a profile and various add-ons. I have done tests using the standard profile with the same result. I am attaching a log where I use Notepad++ as I normally do and where the issue occurs.
[enhanced phonetic reading addon in Notepadplusplus.txt](https://github.com/davidacm/EnhancedPhoneticReading/files/8352661/enhanced.phonetic.reading.addon.in.Notepadplusplus.txt)

**Comment by davidacm** [comment]
This happens when you use the "notepadPlusPlusAccessEnhancement".
If you disable the add-on, or if you use the original notepad plus plus add-on, this add-on work properly.

I think that notepadPlusPlusAccessEnhancement is overriding the speakTextInfo function of my add-on, or is using the original speakTextInfo function.

My add-on works by patching the original speakTextInfo function. But when I use notepadPlusPlusAccessEnhancement, this function of my add-on is not called.

**Comment by DGartmann** [comment]
I can confirm that without the "Notepad++ text's editor : accessibility enhancement" add-on, the phonetic reading add-on works fine. I don't know where I can get the original Notepad++ add-on, nor whether it will work with the NVDA 2022.1 beta. But the add-on "Notepad++ text's editor : accessibility enhancement" has some features that I would like to use. Hope you are able to take it into account. Thanks.
